extensions/chghost-3.2.md

```diff
@@ -5,42 +5,43 @@ copyrights:
   -
     name: "Christine Dodrill"
     period: "2013"
-    email: "xena@yolo-swag.com"
+    email: "me@christine.website"
+  -
+    name: "Ryan"
+    period: "2016"
+    email: "ryan@hashbang.sh"
 ---
-The chghost client capability allows a server to directly inform clients about a
-host or user change without having to send a fake quit and join. This capability
-MUST be referred to as `chghost` at capability negotiation time.
-
-When enabled, clients will get the CHGHOST message to designate the host of a
-user changing for clients on common channels with them.
 
-The CHGHOST message is one of the following:
-
-    :nick!user@host CHGHOST user new.host.goes.here
-
-This message represents that the user identified by nick!user@host has changed
-host to another value. The first parameter is the user of the client. The
-second parameter is the new host the client is using.
+The chghost client capability allows servers to directly inform clients about a
+host or user change without having to send fake quits or joins. This capability
+MUST be referred to as `chghost` at capability negotiation time.
 
-On irc daemons with support for changing the user portion of a client, the
-second form may appear:
+When the capability is enabled, servers MUST send the `CHGHOST` message to
```

Comment:
While I understand what it's trying to say, what it actually says is quite different: "at the time of enabling the capability, server MUST send something to other clients"